docs/support/how-to-capture-and-export-har-file.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Capture and export an HAR file
+description: How to capture an HAR file to help the Octopus team diagnose problems when something unexpected happens.
+position: 17
+---
+
+When something goes wrong in Octopus we may ask you to provide an HAR file to help us diagnose the problem. This file captures a web browser's interaction with a site and may provide insight into where and why a specific request is failing.
+
+:::warning HAR files may contain sensitive data as they include any pages you downloaded while recording and associated cookies. Depending on what has been recorded, this may allow someone with your HAR file to impersonate accounts or any other personal information submitted during recording.
```

Comment:
```suggestion
:::warning
HAR files may contain sensitive data as they include any pages you downloaded while recording and associated cookies. Depending on what has been recorded, this may allow someone with your HAR file to impersonate accounts or any other personal information submitted during recording.
```

Reply:
Fixed this.